src/io/psse.jl

```diff
@@ -601,9 +632,12 @@ function _psse2pm_transformer!(pm_data::Dict, pti_data::Dict, import_all::Bool)
 
                     # Unit Transformations
                     if transformer["CW"] != 1  # NOT "for off-nominal turns ratio in pu of winding bus base voltage"
-                        sub_data["tap"] /= _get_bus_value(bus_id, "base_kv", pm_data)
-                        if transformer["CW"] == 3  # "for off-nominal turns ratio in pu of nominal winding voltage, NOMV1, NOMV2 and NOMV3."
-                            sub_data["tap"] *= transformer["NOMV$m"]
+                        if transformer["CW"] == 2
+                            sub_data["tap"] /= _get_bus_value(bus_id, "base_kv", pm_data)
+                        elseif transformer["CW"] == 3  # "for off-nominal turns ratio in pu of nominal winding voltage, NOMV1, NOMV2 and NOMV3."
+                            if !iszero(transformer["NOMV$m"])
+                                sub_data["tap"] *= transformer["NOMV$m"] / _get_bus_value(bus_id, "base_kv", pm_data)
+                            end
                         end
                     end
```

Comment:
Could you elaborate about what the correct semantics of this should be?

The way the code is organized seems risky to me.  For example, what happens if `transformer["CW"] == 3` and `iszero(transformer["NOMV$m"]) == true`

Reply:
`iszero(transformer["NOMV$m"]) == true` means that the transformer winding `m` nominal voltage is the same as the bus base voltage it is connected to, i.e. `_get_bus_value(bus_id, "base_kv", pm_data)`. So in the case of `transformer["CW"] == 3`, the tap value won't change. For when `transformer["CW"] == 2`, the winding voltage is in kv and does not need to account for `transformer["NOMV$m"]`.

Reply:
Given the complexity of this sepecific section I suggest we re-write the code int he form,

```
if transformer["CW"] == 1
  ...
elseif transformer["CW"] == 2
  ...
elseif transformer["CW"] == 3
  ...
else
  error - unsupported value for "CW" on transformer ....
end
```

Reply:
Recommendation implemented. :)